src/quotamodel.c

```diff
@@ -582,8 +582,8 @@ init_disk_quota_model(uint32 id)
 	memset(&hash_ctl, 0, sizeof(hash_ctl));
 	hash_ctl.keysize   = sizeof(TableSizeEntryKey);
 	hash_ctl.entrysize = sizeof(TableSizeEntry);
-	table_size_map     = DiskquotaShmemInitHash(str.data, INIT_NUM_TABLE_SIZE_ENTRIES, MAX_NUM_TABLE_SIZE_ENTRIES,
-	                                            &hash_ctl, HASH_ELEM, DISKQUOTA_TAG_HASH);
+	table_size_map = DiskquotaShmemInitHash(str.data, MAX_NUM_TABLE_SIZE_ENTRIES, MAX_NUM_TABLE_SIZE_ENTRIES, &hash_ctl,
```

Comment:
now, for each database we allocate map with size 10 * 1024 * 1024 / 100 by default?

Reply:
Yes. How much do we want to reduce memory consumption at the start? Because otherwise we won't be able to use HASH_FIXED_SIZE.

Reply:
So, after all, it is possible to initialize a small shared map at startup, and then increase its size?

Reply:
yes, but then we will need to control the size as it was in the first version of the patch.

Reply:
I think it would better